ClientDependency.Core/FileRegistration/Providers/BaseFileRegistrationProvider.cs

```diff
@@ -24,6 +24,7 @@ public abstract class BaseFileRegistrationProvider : ProviderBase
         protected BaseFileRegistrationProvider()
         {
             EnableCompositeFiles = true;
+            EnableCompositeSingleFiles = false;

```

Comment:
EnableCompositeFiles should possibly do what EnableCompositeSingleFiles does but is probably to big of a change.
EnableCompositeFiles could possibly have been "Enabled"